docs/standards/standard-detection.md

```diff
@@ -13,21 +13,25 @@ The **`interfaceId`** and the **`SupportedStandards:{StandardName}`** key is not
 
 There are two types of **LSP** standards:
 
-- **Interface Standards**: Where we standardize a set of functions. i.e: [LSP0-ERC725Account](./universal-profile/01-lsp0-erc725account.md), [LSP6-KeyManager](./universal-profile/04-lsp6-key-manager.md), [LSP7-DigitalAsset](./nft-2.0/03-LSP7-Digital-Asset.md), etc ..
+- **Interface Standards**: Where we standardize a set of functions. i.e:
 
-- **Metadata Standards**: Where we standardize a set of ERC725Y keys. i.e: [LSP3-UniversalProfile-Metadata](./universal-profile/03-lsp3-universal-profile-metadata.md), [LSP4-DigitalAsset-Metadata](./nft-2.0/02-LSP4-Digital-Asset-Metadata.md), [LSP10ReceivedVaults](./universal-profile/08-lsp10-received-vaults.md), etc.
+  [LSP0-ERC725Account](./universal-profile/01-lsp0-erc725account.md), [LSP6-KeyManager](./universal-profile/04-lsp6-key-manager.md), [LSP7-DigitalAsset](./nft-2.0/03-LSP7-Digital-Asset.md), etc.
 
-![Interface and metadata standards](../../static/img/standard-detection.jpeg)
+- **Metadata Standards**: Where we standardize a set of ERC725Y keys. i.e:
+
+  [LSP3-UniversalProfile-Metadata](./universal-profile/03-lsp3-universal-profile-metadata.md), [LSP4-DigitalAsset-Metadata](./nft-2.0/02-LSP4-Digital-Asset-Metadata.md), [LSP10ReceivedVaults](./universal-profile/08-lsp10-received-vaults.md), etc.
+
+![Interface and metadata standards](/img/standard-detection.jpeg)
 
 These two standards types are fundamental for interacting with smart contracts on the LUKSO blockchain.
 
-**Interface Standard** defines the functions that can be called on a smart contract and their expected parameters. On the other hand, **Metadata Standard** informs about the data set by default on the contract and which keys to query to retrieve such data.
+**Interface Standard** defines the functions that developers can call on a smart contract and their expected parameters. On the other hand, **Metadata Standard** informs about the data set by default on the contract and which keys to query to retrieve such data.
```

Comment:
Please revert to keep it generic, as the functions can be called from Dapps interfaces (frontend + backend) and other smart contracts.